boardgamegeek/api.py

```diff
@@ -888,17 +888,23 @@ def game(self, name=None, game_id=None, choose=BGGChoose.FIRST, versions=False,
         while added_items and len(game.comments) < total:
             page += 1
 
+            params['page'] = page
             xml_root = request_and_parse_xml(self.requests_session,
                                              self._thing_api_url,
-                                             params={"id": game_id,
-                                                     "pagesize": 100,
-                                                     "comments": 1,
-                                                     "page": page})
+                                             params=params,
+                                             timeout=self._timeout,
```

Comment:
I added this and the following two arguments to mimic the `request_and_parse_xml ` call above. I'm not sure if they're necessary, but it seemed like a good idea for the two calls to be done the same way.